SoftLayer/fixtures/SoftLayer_Network_Storage_Hub_Cleversafe_Account.py

```diff
@@ -0,0 +1,39 @@
+credentialCreate = {
+    "accountId": "12345",
+    "createDate": "2019-04-05T13:25:25-06:00",
+    "id": 11111,
+    "password": "nwUEUsx6PiEoN0B1Xe9z9hUCyXMkAFhDOjHqYJva",
```

Comment:
just to confirm... these are not valid anymore right?

Reply:
Yes, this password is not valid, I change the last data before sending it.